harvest/api/_base.py

```diff
@@ -321,7 +333,10 @@ def fetch_option_positions(self):
             - strike_price: Strike price of the option
             - type: 'call' or 'put'
         """
-        raise NotImplementedError("This endpoint is not supported in this broker")
+        debugger.error(
+            f"{type(self).__name__} does not support this broker method: `fetch_option_positions`. Returning an empty list."
+        )
+        return []
```

Comment:
Raising errors are reserved for functions that the expects data from while functions like this we have a suitable null case and can just log a message. Let me know if you don't like this because I am a bit iffy on it too.